compact/hcal/lfhcal.xml

```diff
@@ -0,0 +1,204 @@
+      <fourmodulepositions name="xpos" unit="cm" values="-270 -250 -270 -260 -270 -250 -270 -260 -270 -270 240 260 -10 0 250 260 260 240 260 250 260 260 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 -10 0 30 0 30 -10 -20 -20"/>
+      <fourmodulepositions name="ypos" unit="cm" values="-55 -115 45 -85 -35 105 25 75 -15 5 -115 -55 -265 -265 -85 45 -35 105 25 75 -15 5 255 255 -245 -245 235 235 -225 -225 215 215 -205 -205 195 195 -185 -185 175 175 -165 -165 155 155 -145 -145 135 135 -125 -125 115 115 -105 -105 95 95 -85 -85 75 75 -65 -65 55 55 -45 -45 35 35 -15 -25 5 -25 -15 5"/>
+      <fourmodulepositions name="zpos" unit="cm" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0"/>
```

Comment:
Positions should not be specified as independent x and y lists, and units should be attached to all values not provided as a single field. This should be in the format e.g.:
```
<fourmodulepositions z="0*cm">
  ...
  <position x="-20*cm" y="5*cm"/>
</fourmodulepositions>
```
See mRICH, negative endcap ecal, etc.

Reply:
why would it make a difference? This is a pure formatting choice and would just blow up the file size even further.

Reply:
Modified handing over of positions according to suggestion by Wouter.